module/components/Divider/Readme.md

```diff
@@ -0,0 +1,10 @@
+Divider example:
+
+```js
+<Divider/>
+```
+
+Light Divider example:
+```js
+<Divider light="light" />
```

Comment:
Why are we passing value "light" for property light

Reply:
Done